src/test-gui/java/de/qabel/desktop/ui/LayoutGuiTest.java

```diff
@@ -48,4 +53,20 @@ public void testStuff() throws Exception {
         waitUntil(() -> indicator.getText().equals("1"));
         assertTrue(indicator.isVisible());
     }
+
+    @Test
+    public void testInviteMenuStyle() throws Exception {
+        clickOn(controller.inviteButton);
+        waitUntil(controller.inviteBackground::isVisible);
+        assertTrue(controller.inviteBackground.isVisible());
+        assertEquals(controller.inviteButton.getStyle(), "-fx-effect: innershadow(gaussian, #222222, 10, 10, 10, 10);");
```

Comment:
This would look much nice if you just checked if this button has a specific css class.